cmd/export.go

```diff
@@ -9,32 +9,36 @@ package cmd
 */
 
 import (
-	"errors"
-	"fmt"
-
 	"github.com/spf13/cobra"
+	"github.com/vanilla-os/orchid/cmdr"
 )
 
-func NewExportCommand() *cobra.Command {
-	cmd := &cobra.Command{
-		Example: "apx export htop",
-		Use:     "export <program>",
-		Short:   "Export/Recreate a program's desktop entry from a managed container",
-		RunE:    export,
-	}
-	cmd.Flags().String("bin", "", "Export a binary instead.")
+func NewExportCommand() *cmdr.Command {
+	cmd := cmdr.NewCommand(
+		"export <application/binary>",
+		apx.Trans("export.long"),
+		apx.Trans("export.short"),
+		export,
+	).WithBoolFlag(
+		cmdr.NewBoolFlag(
+			"bin",
+			"",
+			apx.Trans("export.binFlag"),
+			false,
+		),
+	)
+	cmd.Example = "apx export htop"
```

Comment:
Is it possible to add more than one example per command? We could add an example showcasing the `--bin` flag so it's easier to figure out the option exists

Reply:
absolutely, we separate them with \n